dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/search/SearchServiceBean.java

```diff
@@ -792,11 +792,7 @@ public String getLocaleFacetLabelName(String facetLabelName, String facetCategor
     }
 
     private String getDatasetFieldFacetCategoryName(DatasetFieldType matchedDatasetField) {
-        if (matchedDatasetField.isFacetable()) {
-            if (matchedDatasetField.getParentDatasetFieldType() != null) {
-                return matchedDatasetField.getLocaleTitleWithParent();
-            }
-
+        if (matchedDatasetField.isFacetable() && !matchedDatasetField.hasParent()) {
```

Comment:
Not really sure what for there is check for parent field. But in old version check was inverse to what it is now. Is it intentional?

Reply:
Overall logic here was not changed. I only moved code a little to avoid calling `matchedDatasetField.getDisplayName();` twice since we are no longer using `matchedDatasetField.getLocaleTitleWithParent();`.